pyuvdata/uvdata/ms.py

```diff
@@ -1341,12 +1381,12 @@ def write_ms(
 
         self._write_ms_antenna(filepath)
         self._write_ms_data_description(filepath)
-        self._write_ms_feed(filepath)
+        self._write_ms_feed(filepath, pol_order=pol_order)
         self._write_ms_field(filepath)
         self._write_ms_source(filepath)
         self._write_ms_spectralwindow(filepath)
         self._write_ms_pointing(filepath)
-        self._write_ms_polarization(filepath)
+        self._write_ms_polarization(filepath, pol_order=pol_order)
```

Comment:
I think you wouldn't need these `pol_order` arguments to these private methods (or the code to handle them in those methods) at all if you did the `reorder_pols` call before these calls. Am I missing something?

Reply:
I kind of agree, except that you are then calling the reorder function later. If you weren't then I'd totally agree.

One option is to record the order, then reorder, then reorder again to get back to the original order. But if you are calling the reorder method anywhere I don't see a need to handle the ordering in the private methods...

Reply:
I take it back, I mis-read the diff. The reordering that I thought was being done later in the write is actually in the read. I like your approach.